src/recommendations/RecommendationsPage.jsx

```diff
@@ -21,8 +27,10 @@ const RecommendationsPage = ({ location }) => {
   const educationLevel = EDUCATION_LEVEL_MAPPING[location.state?.educationLevel];
   const userId = location.state?.userId;
 
+  const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth - 1 });
```

Comment:
Is this only for extra small devices? Should it not be for small devices too? is this mentioned that the design should be different only for **extra small** devices?

Reply:
yes as per figma, its for extra small devices